arguments/SPIR-V.argdown

```diff
@@ -60,6 +61,11 @@ functionality present in core SPIR-V, like the instructions for ray tracing.
 as well as huge shader code bases. Adding support for another one is something nobody is excited about.
   +> [SPIR-V]
 
+<Limit discord and fragmentation between browsers>:
```

Comment:
Please clarify this point for me to better understand. You are saying that choosing something other than SPIR-V would increase fragmentation between browsers? Why is that?

Reply:
In regards to bulding something new - aka the current Apple approach afaik

Reply:
Ok, thank you for clarifying! I'm afraid, I don't see this point standing strong:
  - browsers pretty much decided already that they are going to independently implement shader translation into MSL/SPIR-V/HLSL. So choosing SPIR-V source or not would not affect this work much, we still don't want to re-use SPIRV-Cross.
  - secondly, the "more subtle differences in implementation, more bugs and limitations" is not a bug, it's a feature of developing a standard, as opposed to developing just a software product. A standard is *validated* by multiple implementations. e.g. Khronos requires multiple implementations of an EXT extension before it's ratified. Bugs are inevitable consequence, but in the end everybody wins from it.

Perhaps, we can land the PR without this block and keep discussing>

Reply:
I have no issue removing the argument if it is not heard.

Reply:
It's not about being heard. It's about an argument that we can't agree on.
The way I see Argdown used is not to split the discussion into arguments that people disagree on. It's the opposite - document and structurize the arguments that the core group *does* agree on, so that everybody, including themselves in the future, can recover it later.
The argument you are making here is not going to be accepted as a valid argument by the core group due to reasons I listed above. So we need to remove it.